index.test.js

```diff
@@ -0,0 +1,48 @@
+import { add, addString, toNumber, filterOrange } from './index';
+
+test('add 1 + 2 to equal 3', () => {
+	expect(add(undefined, 2)).toBe(NaN);
+});
+
```

Comment:
add normal test like below

```js
test('if pass 1, then return 2', () => {

});
```

Reply:
Thank you for your new challenge.
I answered here.

[a6ab08b](https://github.com/nyattsu72/handson-jest/pull/1/commits/a6ab08b7a729a127da9a110528fbbc8940b29be5)